packages/@cdk-cosmos/core/src/ecs-solar-system.ts

```diff
@@ -41,7 +44,29 @@ export class EcsSolarSystemStack extends SolarSystemStack implements EcsSolarSys
   constructor(galaxy: Galaxy, name: string, props?: EcsSolarSystemProps) {
     super(galaxy, name, props);
 
-    const { clusterProps = {}, clusterCapacityProps = {} } = props || {};
+    const { vpc, vpcProps = {}, clusterProps = {}, clusterCapacityProps = {} } = props || {};
+    const { defaultEndpoints = true } = vpcProps;
+
+    // Add endpoints if created Vpc.
+    if (!vpc) {
```

Comment:
Should this be `if (vpc)`...?

Reply:
i only want to add the endpoints if we created the vpc
otherwise its not the responsibility of that component ?

Reply:
Cool...make sense